packages/devtools_app/lib/src/screens/inspector_v2/inspector_controller.dart

```diff
@@ -385,19 +385,14 @@ class InspectorController extends DisposableController
     return _waitForPendingUpdateDone();
   }
 
-  Future<void> refreshInspector() async {
-    // If the user is force refreshing the inspector before the first load has
-    // completed, this could indicate a slow load time or that the inspector
+  Future<void> refreshInspector({bool isManualRefresh = false}) async {
+    // If the user is manually refreshing the inspector before the first load
+    // has completed, this could indicate a slow load time or that the inspector
     // failed to load the tree once available.
-    if (!firstInspectorTreeLoadCompleted) {
-      // We do not want to complete this timing operation because the force
+    if (isManualRefresh && !firstInspectorTreeLoadCompleted) {
+      // We do not want to complete this timing operation because the manual
       // refresh will skew the results.
       ga.cancelTimingOperation(InspectorScreen.id, gac.pageReady);
-      ga.select(
-        gac.inspector,
-        gac.refreshEmptyTree,
-        screenMetricsProvider: () => InspectorScreenMetrics.v2(),
-      );
```

Comment:
why was this removed?

Reply:
Hmm I believe I thought it wasn't capturing what it should, but looking at this now I think this is fine. Added it back in.